src/test/java/org/folio/moduserstest/ExpirationIT.java

```diff
@@ -56,9 +60,45 @@ public void beforeEach() {
   }
 
   @Test
-  void canTriggerExpiration() {
+  void expiredUsersAreDisabled() {
+    final var firstExpiredUser = usersClient.createUser(User.builder()
+      .username("first-user")
+      .active(true)
+      .expirationDate(ZonedDateTime.now().minusHours(3))
+      .build());
+
+    final var secondExpiredUser = usersClient.createUser(User.builder()
+      .username("second-user")
+      .active(true)
+      .expirationDate(ZonedDateTime.now().minusDays(15))
+      .build());
+
     expirationClient.attemptToTriggerExpiration("diku")
       .statusCode(is(HTTP_NO_CONTENT));
+
+    final var firstFetchedUser = usersClient.getUser(firstExpiredUser.getId());
+
+    assertThat(firstFetchedUser.getActive(), is(false));
+
+    final var secondFetchedUser = usersClient.getUser(secondExpiredUser.getId());
+
+    assertThat(secondFetchedUser.getActive(), is(false));
+  }
+
+  @Test
+  void unexpiredUsersIsNotDisabled() {
+    final var unexpiredUser = usersClient.createUser(User.builder()
+      .username("first-user")
+      .active(true)
+      .expirationDate(ZonedDateTime.now().plusHours(3))
+      .build());
+
+    expirationClient.attemptToTriggerExpiration("diku")
+      .statusCode(is(HTTP_NO_CONTENT));
+
+    final var fetchedUser = usersClient.getUser(unexpiredUser.getId());
+
+    assertThat(fetchedUser.getActive(), is(true));
```

Comment:
The tests should not depend on the time zone of the machine that runs the tests:

```
sudo timedatectl set-timezone US/Hawaii
mvn clean install
```

```
[ERROR] Failures:
[ERROR]   ExpirationIT.expiredUsersAreDisabled:81
Expected: is <false>
     but: was <true>
```

```
sudo timedatectl set-timezone Pacific/Auckland
mvn clean install
```

```
[ERROR] Failures:
[ERROR]   ExpirationIT.unexpiredUsersIsNotDisabled:101
Expected: is <true>
     but: was <false>
```

Reply:
@julianladisch Thank you for reporting this.

I'll investigate further as the time zone that is used to set the expiration date should be irrelevant. The user should expire regardless.

Reply:
I don't think I'm able to recreate this. Do you know of a platform agnostic way to do this e.g. by setting the time for the JVM instance that runs for maven?

And I wonder why it didn't fail on the build server which also uses a different time zone.

Reply:
I think I've managed to recreate the issue. I think it is broader than the work in this pull request and will write up an issue describing what I've discovered.

I really appreciate you trying this experiment and reporting this, I think it's uncovered something potentially important.

Reply:
Actually, scratch what I said above.

I'm going to file the issue and raise separate pull request that adds these integration tests and fixes this bug specifically targeted at the mainline. That way these changes aren't mixed together.